### PR TITLE
Show in legend the number of people under each theme

### PR DESCRIPTION
**Pull request by elayabharath**
+ Includes minor refactor of render chart, separate out the computations from render

image

image

### CODE REVIEW

File: lib/interactive.dart [hunk 1]
```diff
@@ -232,13 +238,55 @@ class Interactive {
...
+          SQ_IN_ROW = 24;
+          break;
+        case "idp_status":
+          key = people.idpStatus;
+          SQ_IN_ROW = 24;
+          break;
+        default:
+          logger.error("Selected metric ${_selected.metric} not found");
+      }
+      if (_peopleByLabel[key] != null) {
```
Comment (marianamarasoiu) [inline lib/interactive.dart]: Prefer an early exit here:
```
if (_peopleByLabel[key] == null) return;
...
```
It removes an extra indentation, and avoids accidental errors, e.g. when adding code later on, you don't need to be careful that it's in the `{}` block of the if statement or not.

**Review by marianamarasoiu — APPROVED** [review]
LGTM with one comment, thanks @elayabharath !

File: lib/interactive.dart (hunk 1 above)
Comment (elayabharath) [reply lib/interactive.dart]: Ahh can I use a continue or return in forEach? I think I should start using for(var x in y) pattern than forEach..

Comment (elayabharath) [reply lib/interactive.dart]: Fixed

**Pull request merged**